mmengine/config/utils.py

```diff
@@ -144,3 +149,299 @@ def visit_Assign(self, node):
             return None
         else:
             return node
+
+
+def _is_builtin_module(module_name: str) -> bool:
+    """Check if a module is a built-in module.
+
+    Arg:
+        module_name: name of module.
+    """
```

Comment:
`sys` has an attribute `builtin_module_names` so can we re-use it to simplify the helper function?

Reply:
It seems `sys.builtin_module_name` cannot list all builtin modules:

```bash
In [2]: sys.builtin_module_names
Out[2]: 
('_abc',
 '_ast',
 '_codecs',
 '_collections',
 '_functools',
 '_imp',
 '_io',
 '_locale',
 '_operator',
 '_signal',
 '_sre',
 '_stat',
 '_string',
 '_symtable',
 '_thread',
 '_tracemalloc',
 '_warnings',
 '_weakref',
 'atexit',
 'builtins',
 'errno',
 'faulthandler',
 'gc',
 'itertools',
 'marshal',
 'posix',
 'pwd',
 'sys',
 'time',
 'xxsubtype')
```

As you see, `os` is not in the result.

Reply:
got it.